oauthproxy_test.go

```diff
@@ -660,9 +670,13 @@ func NewPassAccessTokenTest(opts PassAccessTokenTestOptions) *PassAccessTokenTes
 	const emailAddress = "michael.bland@gsa.gov"
 
 	t.opts.SetProvider(NewTestProvider(providerURL, emailAddress))
-	t.proxy = NewOAuthProxy(t.opts, func(email string) bool {
+	var err error
+	t.proxy, err = NewOAuthProxy(t.opts, func(email string) bool {
 		return email == emailAddress
 	})
+	if err != nil {
+		panic(err)
```

Comment:
I was trying to avoid panics in these helper structs and raised errors that any actual tests with a `*testing.T` needed to handle and verify no errors myself.

Want to just leave them for now and I'll handle the cleanup in the msgpack sessions PR?

Reply:
If you've already sorted this in yours, then I'll leave them as panics and you can just clean up when you rebase if that's ok?